winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -9,6 +9,7 @@ app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateS
     });
 
     $scope.numRequiredCompleted = _.sum(_.map(forms, function (form) {
+      console.log(form);
```

Comment:
Nope, these statements will also be tagged by the linter for removal once we turn it on.